lib/perl/Genome/Model/Tools/Picard/CompareSams.pm

```diff
@@ -39,10 +39,10 @@ sub _jar_name {
     return 'CompareSAMs.jar';
 }
 
-sub _java_class_name {
+sub _java_class {
     my $self = shift;
-    return 'net.sf.samtools.apps.CompareSAMs' if $self->use_version eq '1.17';
-    return 'net.sf.picard.sam.CompareSAMs';
+    return qw(samtools apps CompareSAMs) if $self->use_version eq '1.17';
```

Comment:
It is also there at least in [1.16](http://sourceforge.net/p/picard/code/HEAD/tree/tags/1.16/src/java/net/sf/samtools/apps/) (which we call `r116` for some reason).